ipfs-api/src/client/internal.rs

```diff
@@ -789,6 +789,35 @@ impl IpfsClient {
         self.request(request::Commands, None).await
     }
 
+    /// Get and set ipfs config values.
+    ///
+    /// ```no_run
+    /// use ipfs_api::IpfsClient;
+    ///
+    /// let client = IpfsClient::default();
+    /// let res = client.config("Identity.PeerID", None, None, None);
+    /// ```
+    ///
+    #[inline]
+    pub async fn config(
+        &self,
+        key: &str,
+        value: Option<&str>,
+        boolean: Option<bool>,
+        stringified_json: Option<bool>,
+    ) -> Result<response::ConfigResponse, Error> {
```

Comment:
It would simplify function call. What would we name them?

config_set_bool, config_set_json, config_set_string/value ???

I could also split config setting from config getting but that would be 6 functions total.

Let me know!

Reply:
> config_set_bool, config_set_json, config_set_string/value ???

Yep, these sound like good names!

> I could also split config setting from config getting but that would be 6 functions total.

I think I'm fine with adding more functions if they reduce complexity of the API calls, so go ahead with splitting out the getters as well!